Mathlib/SetTheory/Cardinal/Cofinality.lean

```diff
@@ -1215,3 +1215,19 @@ lemma sup_sequence_lt_omega1 {α} [Countable α] (o : α → Ordinal) (ho : ∀
 end Ordinal
 
 end Omega1
+
+namespace Ordinal
+
+theorem isLimit_of_not_isRegular_aleph {o : Ordinal} (h : ¬((aleph o).IsRegular)) :
```

Comment:
I feel like the contraposed statement is more natural, i.e. `aleph o` is regular for non-limit `o`. I think you might agree given that you use `contrapose` in this proof :)

Reply:
What do you think of having them both for convenience? I think I have a different view than mathlib on these things, since I would prefer having lots of one sided versions of `iff` theorems for example, like `Ordinal.pos_of_ne_zero`.

Reply:
I don't think it adds much convenience, since the `contrapose` tactic is always available.

Reply:
Oh wait, `¬ IsRegular` is the spelling we're using for `IsSingular`. Hm, I guess we can have both then